internal/host/bruteforce/modules/telnet.go

```diff
@@ -0,0 +1,170 @@
+package host
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/Method-Security/networkscan/generated/go/bruteforce"
+	"github.com/ziutek/telnet"
```

Comment:
Do we really think this is the best library to use? It only has a single tag, it's a weird license, with a 142 stars. Honestly it feels like a supply chain risk to me more than anything.

Reply:
The other option is https://pkg.go.dev/github.com/reiver/go-telnet@v0.0.0-20180421082511-9ff0b2ab096e but this is bad for the same reason as the above

ill just go and write my own functions that I need

Reply:
As much as I hate having to roll it ourselves, for something like this that is relatively scoped, I think it sometimes is the best option.